### ISSUE TITLE
File Duplicate Issue in submission

### ISSUE THREAD

**Issue by netpacket** [cuckoosandbox/cuckoo#1678]
I am submitting a zip with folder with three files marked as duplicates on the web gui. I am only aware of --unique option on submission: cuckoo submit --unique. I have not added this. 

 I have never uploaded these files before. Is there config file that turns on a check for duplicated files? Or, is this have anything do with VirusTotal module? 

I have never seen this check.

**Comment by doomedraven** [comment]
it just files with the same hash, so it aware you yo save your resources and time ;)

**Comment by netpacket** [comment]
![file_duplication](https://user-images.githubusercontent.com/27368535/27559826-d2252a10-5a76-11e7-9298-c6d2c5fe7448.png)
I took the md5sum of all the files inside of the zip. I am getting different hashes on each file.

**Comment by netpacket** [comment]
Is it possible Cuckoo thinks that the files have the same hash because the files are inside subfolder?

**Comment by jbremer** [comment]
No, it's actually because Cuckoo thinks the files are empty. Which is most likely due to being password protected, which isn't very well supported as of yet (unfortunately). We should be addressing that soon'ish.
